clouddriver-kubernetes/src/main/groovy/com/netflix/spinnaker/clouddriver/kubernetes/v1/deploy/description/securitygroup/KubernetesSecurityGroupDescription.groovy

```diff
@@ -43,7 +46,7 @@ class KubernetesIngressBackend {
 
 @AutoClone
 @Canonical
-class KubernetesIngressTls {
+class KubernetesIngressTlS {
```

Comment:
Any reason why the `S` was capitalized?

Reply:
OCD I guess ;) it seems to be capitalised everywhere else. I can revert it if necessary